{"labels": [{"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "TestLoginP"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "host", "value": "G50-75M"}, {"name": "thread", "value": "15668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login_pytest"}], "historyId": "0cd1a3fa99d0599dc61cd043dcb6fb74", "fullName": "TestCases.test_login_pytest.TestLoginP#test_login_0_failed_by_wrong_datas[data4]", "uuid": "fe51a11b-3ba8-407c-b57c-e9fe78084bc5", "start": 1561889727002, "name": "test_login_0_failed_by_wrong_datas[data4]", "status": "passed", "parameters": [{"name": "data", "value": "{'user': '1868472 0553', 'password': 'python', 'check': '请输入正确的手机号'}"}], "stop": 1561889727711}